src/ddspdrum/torchmodparameter.py

```diff
@@ -0,0 +1,97 @@
+"""
+Parameter Class
```

Comment:
What is this module for? Looks like TorchSynthModule is still using the other ModParameters?

Reply:
I see now that the plan is for this to inherit from nn.Parameter

Reply:
you're right tho we can remove it